test/fuzz/cbor/cbor_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cbor_test
+
+import (
+	"testing"
+
+	"k8s.io/kubernetes/test/fuzz/cbor"
+)
+
+// FuzzDecodeAllocations wraps the FuzzDecodeAllocations go-fuzz target as a "go test" fuzz test.
+func FuzzDecodeAllocations(f *testing.F) {
+	f.Add([]byte("\xa2\x4aapiVersion\x41x\x44kind\x41y")) // {'apiVersion': 'x', 'kind': 'y'}
+	f.Fuzz(func(t *testing.T, in []byte) {
+		defer func() {
+			if p := recover(); p != nil {
+				t.Fatal(p)
```

Comment:
Does a failure result in enough information to reproduce?  (Does go test fuzz automatically output the fuzzed input that resulted in the failure?)

Reply:
Yes, it'll actually write the input to a file under testdata. Also, all the inputs saved there are exercised when it's run with a normal "go test" invocation, to provide regression coverage.

This PR includes one example of a failing input that is fixed in the latest library version (https://github.com/fxamacker/cbor/pull/533/). Its diagnostic notation is `{h'3030303030303030B030': [[h'', h'', h'', h'', h'', h'', h''], h'', h'', h'', h'', h''], h'': -17}`.

Reply:
Great!  Thanks.